package.json

```diff
@@ -94,7 +94,8 @@
     "@mui/lab": "^5.0.0-alpha.82",
     "@mui/material": "^5.6.2",
     "@mui/styles": "^5.8.0",
-    "@mui/x-date-pickers": "^6.6.0",
+    "@mui/x-data-grid": "^6.18.6",
+    "@mui/x-date-pickers": "^6.18.7",
```

Comment:
These were only added for resource table in PoC. Once we play the resource table tanstack story (probably hasn't even been written yet) these dependencies can be removed